test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1525,11 +1525,6 @@ class test_ucp_am_nbx_rndv_dts : public test_ucp_am_nbx_rndv {
 
         test_ucp_am_nbx::cleanup();
     }
-
-    virtual unsigned enable_proto()
-    {
-        return 0;
-    }
 };
```

Comment:
why was it incorrect?

Reply:
Enabling/disabling new protocols should be done inside ctor.
Thus the method should be called inside ctor. Making it virtual may lead to not obvious behavior.

Reply:
These changes was merged within https://github.com/openucx/ucx/pull/8131